### PR TITLE
vip-plugins: "VIP Featured Plugins" -> "VIP Featured Partners"

### PR DESCRIPTION
**Pull request by curtisbelt**
WordPress VIP doesn't have an approved list of plugins, and therefore no Featured Plugins.

It does, however, have a list of [Featured Partners](http://wpvip.com/partners), so this PR updates the title in the admin page shown in all customer WPVIP sites to reflect that.

image

### CODE REVIEW

File: vip-plugins/vip-plugins.php [hunk 1]
```diff
@@ -56,15 +56,15 @@ function wpcom_vip_render_vip_featured_plugins() {
 	if ( empty( $plugins ) ) {
 		?>
 		<div class="notice notice-error">
-			<p><?php _e( 'Unable to display VIP featured plugins; try refreshing this page in a few minutes. If this error persists, please contact VIP Support.', 'vip-plugins-dashboard' ); ?></p>
+			<p><?php _e( 'Unable to display VIP featured partners; try refreshing this page in a few minutes. If this error persists, please contact VIP Support.', 'vip-plugins-dashboard' ); ?></p>
```
Comment (GaryJones) [inline vip-plugins/vip-plugins.php:59]:
```suggestion
			<p><?php esc_html_e( 'Unable to display VIP featured partners; try refreshing this page in a few minutes. If this error persists, please contact VIP Support.', 'vip-plugins-dashboard' ); ?></p>
```

File: vip-plugins/vip-plugins.php [hunk 2]
```diff
@@ -56,15 +56,15 @@ function wpcom_vip_render_vip_featured_plugins() {
...
 		</div>
 		<?php
 		return;
 	}
 
 	?>
 	<div class="featured-plugins notice">
-		<h3><?php _e( 'VIP Featured Plugins', 'vip-plugins-dashboard' ); ?></h3>
+		<h3><?php _e( 'VIP Featured Partners', 'vip-plugins-dashboard' ); ?></h3>
```
Comment (GaryJones) [inline vip-plugins/vip-plugins.php:67]:
```suggestion
		<h3><?php esc_html_e( 'VIP Featured Partners', 'vip-plugins-dashboard' ); ?></h3>
```

**Comment by brandon-m-skinner** [comment]
I think the name should be something like VIP Technology Partner Plugins. Thoughts?

**Comment by ryansholin** [comment]
:wave: VIP "Featured Technology Partners" is correct, thanks! (Not all partners have plugins; not all partners are featured; this is intentional.)

**Review by brandon-m-skinner — APPROVED** [review]

File: vip-plugins/vip-plugins.php [hunk 3]
```diff
@@ -56,15 +56,15 @@ function wpcom_vip_render_vip_featured_plugins() {
 	if ( empty( $plugins ) ) {
 		?>
 		<div class="notice notice-error">
-			<p><?php _e( 'Unable to display VIP featured plugins; try refreshing this page in a few minutes. If this error persists, please contact VIP Support.', 'vip-plugins-dashboard' ); ?></p>
+			<p><?php esc_html_e( 'Unable to display VIP featured partners; try refreshing this page in a few minutes. If this error persists, please contact VIP Support.', 'vip-plugins-dashboard' ); ?></p>
```
Comment (brandon-m-skinner) [inline vip-plugins/vip-plugins.php:59]:
```suggestion
			<p><?php esc_html_e( 'Unable to display VIP Featured Technology Partners; try refreshing this page in a few minutes. If this error persists, please contact VIP Support.', 'vip-plugins-dashboard' ); ?></p>
```

File: vip-plugins/vip-plugins.php [hunk 4]
```diff
@@ -56,15 +56,15 @@ function wpcom_vip_render_vip_featured_plugins() {
...
 		</div>
 		<?php
 		return;
 	}
 
 	?>
 	<div class="featured-plugins notice">
-		<h3><?php _e( 'VIP Featured Plugins', 'vip-plugins-dashboard' ); ?></h3>
+		<h3><?php esc_html_e( 'VIP Featured Partners', 'vip-plugins-dashboard' ); ?></h3>
```
Comment (brandon-m-skinner) [inline vip-plugins/vip-plugins.php:67]:
```suggestion
		<h3><?php esc_html_e( 'VIP Featured Technology Partners', 'vip-plugins-dashboard' ); ?></h3>
```

**Review by brandon-m-skinner — CHANGES_REQUESTED** [review]
Added Technology to the title as per @ryansholin s comment.

**Review by brandon-m-skinner — APPROVED** [review]

**Comment by brandon-m-skinner** [comment]
r1269-stacks

**Pull request merged**